sdk/storage/azure-storage-file/azure/storage/file/file_client.py

```diff
@@ -1112,45 +1111,70 @@ def list_handles(self, **kwargs):
             page_iterator_class=HandlesPaged)
 
     @distributed_trace
-    def close_handles(
-            self, handle=None, # type: Union[str, HandleItem]
-            **kwargs # type: Any
-        ):
-        # type: (...) -> Any
-        """Close open file handles.
-
-        This operation may not finish with a single call, so a long-running poller
-        is returned that can be used to wait until the operation is complete.
+    def close_handle(self, handle, **kwargs):
+        # type: (Union[str, HandleItem], Any) -> int
+        """Close an open file handle.
 
         :param handle:
-            Optionally, a specific handle to close. The default value is '*'
-            which will attempt to close all open handles.
+            A specific handle to close.
         :type handle: str or ~azure.storage.file.Handle
         :keyword int timeout:
             The timeout parameter is expressed in seconds.
-        :returns: A long-running poller to get operation status.
-        :rtype: ~azure.core.polling.LROPoller
+        :returns:
+            The number of handles closed (this may be 0 if the specified handle was not found).
```

Comment:
handle :grinning:

Reply:
Actually I think handles is correct in this case... although the result should only be 1 or 0...

Reply:
Sure! as long as it is intended! I just thought it will never be a plural 😀